contracts/staking.tmy/src/staking.tmy.cpp

```diff
@@ -239,6 +239,11 @@ namespace stakingtoken
             });
 
             total_yield += yield;
+         } 
+         else if (itr->unstake_requested && eosio::current_time_point() >= itr->unstake_time + eosio::days(5) ) 
```

Comment:
1. use the contract constants for the release time
2. calculate `eosio::current_time_point()` out of the loop for efficiency
3. don't think you need to check `itr->unstake_requested` as we already know it is not `!itr->unstake_requested` from the if condition